repose-aggregator/services/service-client/api/src/main/java/org/openrepose/core/services/serviceclient/akka/AkkaServiceClient.java

```diff
@@ -29,6 +29,8 @@
  */
 public interface AkkaServiceClient {
 
+    void setConnectionPoolId(String connectionPoolId);
```

Comment:
also, also wik. This is internal api changes so it's only minor version number. major is reserved when we are forcing people to change their configs or operating environment (java 8).